templates/wg.conf.j2

```diff
@@ -68,3 +68,26 @@ SaveConfig = true
     {% endif %}
   {% endif %}
 {% endfor %}
+{% if wireguard_unmanaged_peers is defined %}
+
+    # Peers not managed by ansible from wireguard_unmanaged_peers
+    {% for peer in wireguard_unmanaged_peers.keys() %}
+    [Peer]
+    # {{ peer}}
+    {% if wireguard_unmanaged_peers[peer].public_key is defined %}
```

Comment:
For public key can drop the if as it's required and not valid peer without it. 

I'd keep others optional although allowed ips is effectively required but not by syntax validation. Maybe someone would update allowed ips dynamically somehow externally. No harm in generating non-usable but valid config.